docs/AboutUs.md

```diff
@@ -32,45 +33,16 @@ You can reach us at our individual emails below!
   * Integration
```

Comment:
I think we can change this to match the rest of the formats:
```
Roles:
Responsibilities:
```

Reply:
Sounds good, I have just made the necessary changes!